### PR TITLE
[#1362] - Marketing button Test

### LINKED ISSUE #1362: UNIT TESTS - components/MarketingButton
## What

Create unit tests using [Jest](https://jestjs.io/en/) and [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/).

Here are a couple examples of how unit tests should be created:

1. [Unit tests for our A component](https://github.com/thenewboston-developers/Website/blob/development/src/components/A/A.test.tsx)
2. [Unit tests for Icon component](https://github.com/thenewboston-developers/ui-js/blob/master/src/components/Icon/Icon.test.tsx)
3. [Unit tests for Button component](https://github.com/thenewboston-developers/ui-js/blob/master/src/components/Inputs/Button/Button.test.tsx)

## General Acceptance Criteria for writing Unit tests

1. Should create a `[nameOfComponent].test.tsx` within the same directory of the component.
2. Should have tests that tests every `prop` of the component.
3. If there are any conditional logic within the component, there should be unit tests that accounts for each conditional branches.
4. `npm run test` should be passing.

## Specific Acceptance Criteria

1. make sure you tests for each prop

### PR DESCRIPTION
**Pull request by denistsoi**
Resolves #1362.
Acc: baa336bca22c7590256a86c002e4dbe6c1f1c1f6c0a0fd3f0681eecb2cad0787

### CODE REVIEW

**Comment by angle943** [comment]
Tests look good, but could you add one for the className prop?

File: src/components/MarketingButton/MarketingButton.test.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+describe('MarketingButton component', () => {
+  test('renders with expected props of discord as website', () => {
+    render(<MarketingButton website={SocialMedia.discord} />);
+
+    const el = screen.getByTestId('A');
+    expect(el).toBeTruthy();
+    expect(el.hasAttribute('href')).toBeTruthy();
+    expect(el.getAttribute('href')).toEqual(socialMediaUrls[SocialMedia.discord]);
+
+    expect(el.className).toContain('MarketingButton');
```
Comment (denistsoi) [inline src/components/MarketingButton/MarketingButton.test.tsx:20]: added className test

Comment (angle943) [reply src/components/MarketingButton/MarketingButton.test.tsx:20]: @denistsoi lets keep them as unit tests. Each test should ideally be testing for one thing.

Please add:

1. One test with title 'renders correctly with default classNames', which tests for the existence of `MarketingButton` class and `MarketingButton__img` class.
2. One test with the title `renders correctly with className passed in`, with the prop `className: 'test-class'`, which tests for the existence of `test-class` className and `test-class__img` className for the img.

**Review by angle943 — CHANGES_REQUESTED** [review]
please see my comment above

**Comment by angle943** [comment]
@denistsoi added a test for the `img` element's classnames

**Review by angle943 — APPROVED** [review]

**Pull request merged**